kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KubernetesApplicationOperation.scala

```diff
@@ -101,8 +102,22 @@ class KubernetesApplicationOperation extends ApplicationOperation with Logging {
           debug(s"Successfully got application info by $tag: $info")
           info
         } else {
-          // client mode
-          jpsOperation.getApplicationInfoByTag(tag)
+          // Kyuubi should wait second if pod is not be created
+          submitTime match {
```

Comment:
for complex logical, we can just `return` in `if` branch, and save an `else` block.

Reply:
Warn log should also be required by users